tests/python/contrib/test_hexagon/rpc/test_launcher.md

```diff
@@ -26,20 +26,38 @@ Here are the steps that are taken to prepare a runtime on a Hexagon device to te
 - Build TVM library with Hexagon support for host machine.
 - Build TVMRuntime library and C++ RPC server for host machine.
 
-To build these pieces, you can use a cmake command as follow.
+Note: before moving forward make sure to export Clang libraries to `LD_LIBRARY_PATH` and Hexagon toolchain to `HEXAGON_TOOLCHAIN`.
```

Comment:
A few suggestions:
- nitpick: "before moving forward" is an unusual phrasing in a file like this, and parsing it slowed down my reading a little bit.  It might be more idiomatic to say something like, "First, ensure that ...".

- For variables like these, sometimes people get confused regarding how deep into a directory structure the variables should point.  It might help to give concrete examples to clarify that.

Reply:
added more details. thanks!